client/packages/system/src/Stock/ListView/AppBarButtons.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react';
+import {
+  DownloadIcon,
+  useNotification,
+  AppBarButtonsPortal,
+  Grid,
+  useTranslation,
+  FileUtils,
+  LoadingButton,
+  SortBy,
+  EnvUtils,
+  Platform,
+} from '@openmsupply-client/common';
+import { StockLineRowFragment, useStock } from '../api';
+import { stockLinesToCsv } from '../../utils';
+
+export const AppBarButtonsComponent: FC<{
+  sortBy: SortBy<StockLineRowFragment>;
+}> = ({ sortBy }) => {
+  const { success, error } = useNotification();
+  const t = useTranslation(['distribution', 'common']);
+  const { fetchAsync, isLoading } = useStock.line.listAll(sortBy);
```

Comment:
Small thing, not sure what the correct answer is, but I'm not sure the "Export" should necessarily use the sort order of the current view. It doesn't respect other aspects of the current display (such as the filtering), so it's possibly inconsistent/misleading to export something different depending on the current displayed sort.

I would think that just exporting by "name" sorting would be okay. It also means that if you make different exports on different days, you can compare your lists much easier. And if they need any additional sorting/filtering they can just their spreadsheet app for that.

Reply:
it seemed like a cute idea at the time - though like you say, the pagination and filtering isn't used when exporting so it does make the sort inconsistent. I'll run through and update them all.